distributednetworkmanager/src/main/java/com/gruppo1/distributednetworkmanager/KBucket.java

```diff
@@ -1,56 +1,64 @@
 package com.gruppo1.distributednetworkmanager;
 
-/**
- * @author
- */
-public class KBucket extends Bucket<PeerNode> {
+
+import android.content.Context;
+
+import java.security.acl.NotOwnerException;
+
+public class KBucket extends Bucket<Node> {
+
     private int dimension;
```

Comment:
"dimension" is present also in Bucket<Node> so it isn't necessary to insert it and just add a getDimension() on Bucket<Node> and on the constructor of KBucket invoke "super(dim)".